v03_pipeline/lib/reference_data/compare_globals.py

```diff
@@ -96,6 +91,16 @@ def get_datasets_to_update(
         datasets_to_update.update(
             ht1_globals[field.name].keys() ^ ht2_globals[field.name].keys(),
         )
+        if field.name == 'enums':
```

Comment:
good catch on these weird enum fields.  I think it might be better to just exclude these when we construct the globals [here](https://github.com/broadinstitute/seqr-loading-pipelines/pull/723/files#diff-2460cc720d61d0c78e554d81ddfcd480509c4b260270709c2952a6bcb091d729R70)?  they're basically all non-reference-data related so can be considered separate from this process we're doing to compare reference data globals.

Reply:
Actually, won't `from_ht` above handle this by filtering to `datasets`?

Reply:
(one hint here, you can add a `for dataset in datasets:` loop to `from_ht` which'll make it look similar to `from_dataset_configs` and it'll be super nice!

Reply:
In `from_ht`, after I took out [these lines](https://github.com/broadinstitute/seqr-loading-pipelines/pull/723/files/2df1a3f7232386df9fafd63253b3d14b8923a214#diff-2460cc720d61d0c78e554d81ddfcd480509c4b260270709c2952a6bcb091d729L72-L76), only the `selects` are filtered to `datasets` using the `for dataset in datasets:` loop.
I prefer not to filter the datasets in `from_ht` at all because any extra dataset in these globals that isn't one of those special annotation enums is a dataset that needs to be dropped needs to be dropped.

Reply:
oof, yah that's correct about the dropping 🤔.  What about filtering `enums` to only the datasets that are in `paths` or `versions`?  not sure if that's better.

Reply:
I could also filter `enums` to `datasets` so we'd still catch extra datasets in `paths` and `versions`

Reply:
👍 I don't think there should be anything in paths/versions that's not in enums.  But that's basically just a heuristic and not enforced.